docs/12_0_0/components/column.md

```diff
@@ -22,6 +22,7 @@ treetable and more.
 | ariaHeaderText | null | String | Label to read by screen readers, when not specified headerText is used.
 | caseSensitiveSort         | false              | Boolean          | Case sensitivity for sorting, insensitive by default.
 | colspan | 1 | Integer | Defines the number of columns the column spans.
+| converter | null | Converter/String | An el expression or a literal text that defines a converter for the component. When it’s an EL expression, it’s resolved to a converter instance. In case it’s a static text, it must refer to a converter id
```

Comment:
EL always upper case

Reply:
I copied it. Was planning to find and replace it everywhere in a new PR. If I would do it now, it would pollute this PR.